src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/DataSetFilesControllerTests.cs

```diff
@@ -1600,8 +1600,19 @@ public async Task DataSetFileMetaCorrectlyReturned_Success()
                         .WithType(FileType.Data)
                         .WithDataSetFileMeta(_fixture.DefaultDataSetFileMeta()
                             .WithGeographicLevels(["National", "Local authority"])
-                            .WithTimeIdentifier(TimeIdentifier.AcademicYear)
-                            .WithYears([2000, 2001, 2002])
+                            .WithTimePeriodRange(new TimePeriodRangeMeta
+                            {
+                                Start = new TimePeriodRangeBoundMeta
+                                {
+                                    TimeIdentifier = TimeIdentifier.AcademicYear,
+                                    Period = "2000",
+                                },
+                                End = new TimePeriodRangeBoundMeta
+                                {
+                                    TimeIdentifier = TimeIdentifier.AcademicYear,
+                                    Period = "2002",
+                                },
+                            })
```

Comment:
Would be good to have some generator extensions for this to make this a bit nicer e.g. something like this:

```cs
_fixture.DefaultTimePeriodRangeMeta()
  .WithStart("2000", TimeIdentifier.AcademicYear)
  .WithEnd("2002", TimeIdentifier.AcademicYear)
```

Similar thing applies in the other test case further down